tests/integration/defs/perf/disagg/utils/config_loader.py

```
@@ -0,0 +1,551 @@
+            config_data: Full YAML config data
+
+        Returns:
+            MetricsConfig (default or merged with overrides)
+        """
+        # Get default configuration
+        config_key = (test_type, test_category)
+        default_config = DEFAULT_METRICS_CONFIG.get(config_key)
+        if not default_config:
+            # If no default configuration, trigger exception
+            logger.warning(f"No default metrics config for config_key: {config_key}")
+            raise ValueError(f"No default metrics config for config_key: {config_key}")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)